RecoParticleFlow/PFProducer/plugins/PFLinker.cc

```diff
@@ -77,6 +78,22 @@ class PFLinker : public edm::stream::EDProducer<> {
 
 DEFINE_FWK_MODULE(PFLinker);
 
+void PFLinker::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<std::vector<edm::InputTag>>("PFCandidate", {edm::InputTag("particleFlow")});
+  desc.add<edm::InputTag>("GsfElectrons", {"gedGsfElectrons"});
+  desc.add<edm::InputTag>("Photons", {"gedPhotons"});
+  desc.add<edm::InputTag>("Muons", {"muons", "muons1stStep2muonsMap"});
+  desc.add<bool>("ProducePFCandidates", true);
+  desc.add<bool>("FillMuonRefs", true);
+  desc.add<std::string>("OutputPF", "");
+  desc.add<std::string>("ValueMapElectrons", "electrons");
+  desc.add<std::string>("ValueMapPhotons", "photons");
+  desc.add<std::string>("ValueMapMerged", "all");
+  desc.add<bool>("forceElectronsInHGCAL", false);
+  descriptions.add("pfLinker", desc);
```

Comment:
No reason in concrete. Just tried to keep the same python configuration file names as the existing ones

Reply:
in this case (and below) using `addWithDefaultLabel` will keep the same name as the existing python configuration file.

Reply:
I had the impression that using addWithDefaultLabel would change the python filename name to the same as the c++ filename (i.e. in this case PFLinker.py instead of pfLinker_cfi.py) and also the internal module would change to PFLinker() instead of pfLinker.
But I am fine to change these two files, properly propagating the names to the files that are importing this module

Reply:
> (i.e. in this case PFLinker.py instead of pfLinker_cfi.py) 

no, it will create `pfLinker_cfi.py` not `PFLinker.py`.

Reply:
Addressed in a848a6c0ac22d23ae66d13f7c993f3c7ecdb1a15